code/lib/test/template/stories/module-mocking.stories.ts

```diff
@@ -0,0 +1,28 @@
+import { global as globalThis } from '@storybook/global';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore This alias is set in the sandbox. Using ts-ignore instead of ts-expect-error to avoid build errors in the sandbox.
+// eslint-disable-next-line import/no-unresolved
+import { foo } from '#utils';
+import { expect, fn, isMockFunction, mocked } from '@storybook/test';
+
+export default {
+  component: globalThis.Components.Button,
+  args: {
+    onClick: fn(),
+    label: 'Mock story',
+  },
+  parameters: {
+    chromatic: {
+      disable: true,
+    },
+  },
+  loaders: () => {
```

Comment:
is loaders used on purpose, or did you mean to use beforeEach?

Reply:
beforeEach was not available when I made this story! will change :)